### ISSUE TITLE
Russian symbols and time in console

### ISSUE THREAD

**Issue by Raingerlina** [CyberdyneCC/Thermos#562]
Hello, I receive strange symbols in console instead of russian symbols. And, what's more, it show time in very strange way. It is surely not chat plugin fault, cause I tried start server without plugins at all.
http://pastebin.com/30STXrjR

It is easy to recreate mistake, just type something in russian. Here are some symbols for example: йцукен.

Mods Installed: not needed
Plugins Installed: not needed
Thermos Version: Build 58
Forge Version: 1614

**Comment by Robotia** [comment]
Is your server running on Windows?

**Comment by Raingerlina** [comment]
No, Linux. At least support of service, where I rent server, said me that :)

**Comment by Robotia** [comment]
Where are you typing this? Is it a web control panel? SSH?
Can you show screenshot?

**Comment by Raingerlina** [comment]
![screenshot_2016-10-29-09-23-00-051_com android chrome](https://cloud.githubusercontent.com/assets/23079606/19827619/7b31b5d6-9db9-11e6-8f27-334a8e5e9ae3.png)
Used 'braodcast' with russian symbols for example

**Comment by Raingerlina** [comment]
I need to add, that when anyone types in chat something, in-game it looks acceptable, but in console I see this codes. 
![image](https://cloud.githubusercontent.com/assets/23079606/19832952/523c0830-9e3a-11e6-9c08-3dd00c981aa6.png)
![image](https://cloud.githubusercontent.com/assets/23079606/19832955/6a71aa9a-9e3a-11e6-9c6a-a9a20d769164.png)

**Comment by Robotia** [comment]
In this case, it seems the web panel character encoding is at fault.

Can you contact the provider and ask them to allow html entity display in the console?
The &#...; correspond to these: http://symbolcodes.tlt.psu.edu/bylanguage/cyrchart.html

But are not being converted

**Comment by Raingerlina** [comment]
Okay, I got it, thank you for response. I'll try to ask provider, hope, it will help :)
